dev/protos/firestore/bundle.proto

```diff
@@ -79,6 +79,9 @@ message BundledDocumentMetadata {
 
   // Whether the document exists.
   bool exists = 3;
+
+  // The query name in this bundle, if the document is added via a named query.
+  string query = 4;
```

Comment:
A document can match multiple queries and should be a list that is merged with each update, shouldn't it?